app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -297,7 +297,7 @@ class Result extends React.Component {
         this.setState({annotations: newAnnotations});
       }
     }
-    update_annotation_text(annotation.id, marked(annotation.content, {sanitize: true}));
+    update_annotation_text(annotation.annotation_text_id, annotation.content, annotation.id);
```

Comment:
Leave the `marked` call (you might have removed it because it's being deprecated, but we should keep it for now).

Reply:
Actually the marked call causes the annotation text hover to be padded with HTML (like if my annotation.content said 'hi', calling marked on it would produce <p>hi</p> and this will also get displayed in the annotation hover exactly (with the HTML tags as text)